green/runner.py

```diff
@@ -48,30 +54,45 @@ def run(suite, stream, args):
 
         result.startTestRun()
 
-        tests = toProtoTestList(suite)
+        tests = [_AsyncChunk(t, multiprocessing.Manager().Queue()) for t in toParallelTestTargets(suite, args.targets)]
```

Comment:
I have noticed that creating tons of queues like this can be *very* slow, to the point where there's about four or five seconds of overhead for green's own test suite. Is there a better container that we could be using here? Should it be redesigned to only use a single queue?